db/models/orders.js

```diff
@@ -86,27 +107,37 @@ const getAllOrdersByUser = async ({ username }) => {
           `,
       [username]
     );
-
+    if (!orders) {
+      throw {
+        name: "NoExistingInformation",
+        message: "No orders currently exist.",
```

Comment:
What if the user hasn't placed any orders before? this will still run. Maybe we change the message to display something to relay that i.e. message: "You haven't placed any orders"

Reply:
Done.